build.gradle

```diff
@@ -255,7 +255,8 @@ dependencies {
     compile 'com.github.broadinstitute:picard:' + picardVersion
     externalSourceConfiguration 'com.github.broadinstitute:picard:' + picardVersion + ':sources'
     compile ('org.genomicsdb:genomicsdb:' + genomicsdbVersion)  {
-        exclude module: 'log4j'
+        exclude module: 'log4j-api'
+        exclude module: 'log4j-core'
```

Comment:
@lbergelson Can you comment on this change to the exclusions for the GenomicsDB dependency? Do you foresee any issues as a result of this change?

Reply:
I don't think this will be a problem but I'm not 100% sure I know why this was necessary.  I assume this was driven by something practical that came up. I assume we're excluding this because it conflicts with our desired log4j version.  Did genomics db bump their version to use the log4j2 api instead of the original log4j?  

@nalinigans Can you comment on this?

Reply:
@lbergelson, @droazen, yes GenomicsDB bumped up to [log4j2](https://github.com/GenomicsDB/GenomicsDB/pull/79) which necessitated the api and core exclusions.